lib/common.mk

```diff
@@ -38,12 +38,12 @@ LIB_OBJS += $(foreach obj,$(UTIL_OBJS),$(LIB_DIR)/util/$(obj))
 EXTRA_DEPS +=
 EXTRA_USER_DEPS +=
 
-LDFLAGS+=-L$(LIBXDP_DIR)
+LDFLAGS+=-L$(LIBXDP_DIR)/$(OBJDIR)
```

Comment:
This implies that OBJDIR is always relative to LIBXDP_DIR - I am not sure there is any such guarantee? I.e., OBJDIR could be an absolute path?

How were you setting OBJDIR, exactly, when you ran into this?

Reply:
Indeed I'm setting `OBJDIR=output`.

Will take care both both cases OBJDIR is absolute and relative.